concert/storage.py

```diff
@@ -581,7 +581,13 @@ async def write_sequence(self, name: Optional[str] = "") -> None:
         :param path: path to write to
         :type path: str
         """
-        await self.device.write_sequence(name)
+        import tango
+        self.device.command_inout_asynch("write_sequence", name, True)
+        while await self.device.state() == tango.DevState.RUNNING:
+            await asyncio.sleep(0.1)
+        if await self.device.state() != tango.DevState.STANDBY:
+            raise StorageError("Error within TangoRemoteWriter")
```

Comment:
@MarcusZuber this does not work, I get errors coming from the `FAULT` state for no reason and what's worse, on the Tango server side I don't see the exception which caused it, even though you re-raise there. We should revisit why you chose this implementation and perhaps simplify it.